docs/UserGuide.md

```diff
@@ -131,24 +137,7 @@ Examples:
 * 'edit 2 n/A Brief History of Time e/abhot@gmail.com' Edits the name and contact email address of the 2nd book to be A Brief History of Time and abhot@gmail.com respectively.
 * 'edit 3 p/Scribner Publisher t/' Edits the publisher of the 3rd book to be Scribner Publisher and clears all existing tags.
 
-### Locating persons by name: `find`
```

Comment:
It seems that the documentation of this command is from the original project. Maybe it can be changed according to the current implementation.